core/src/test/java/org/hibernate/ogm/backendtck/jpa/JPAStartWithoutProvider.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.backendtck.jpa;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.util.HashMap;
+import java.util.Map;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.EntityTransaction;
+import javax.persistence.Persistence;
+
+import org.hibernate.ogm.utils.PackagingRule;
+import org.hibernate.ogm.utils.TestForIssue;
+import org.hibernate.ogm.utils.TestHelper;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/**
+ * Tests a store and load scenario, using the JPA interface,
+ * without defining a persistence provider to enable Hibernate OGM engine.
+ *
+ * @author Fabio Massimo Ercoli
+ */
+@TestForIssue(jiraKey = "OGM-1517")
+public class JPAStartWithoutProvider {
+
+	private static final String POEM_NAME = "Orlando Furioso";
+
+	@Rule
+	public PackagingRule packaging = new PackagingRule( "persistencexml/ogm-noprovider.xml", Poem.class );
+
+	@Rule
+	public ExpectedException thrown = ExpectedException.none();
+
+	@Test
+	public void testJPAStartWithoutProvider() {
+		Map<String, Object> settings = new HashMap<>( TestHelper.getDefaultTestSettings() );
+
+		EntityManagerFactory factory = null;
+		try {
+			factory = Persistence.createEntityManagerFactory( "ogm", settings );
+
+			EntityManager manager = null;
+			try {
+				manager = factory.createEntityManager();
+				createAndLoadAPoem( manager );
+			}
+			finally {
+				if ( manager != null ) {
+					manager.close();
+				}
+			}
+		}
+		finally {
+			if ( factory != null ) {
+				factory.close();
+			}
```

Comment:
I think that for this test I would prefer to close everything in a `@After` method. This way the test becomes much smaller and easier to follow. This is just a suggestion.